block_darwin.go

```diff
@@ -178,6 +178,15 @@ func (ctx *context) makePartition(disk, s diskOrPartitionPlistNode, isAPFS bool)
 	}, nil
 }
 
+func busTypeFromBusProtocol(busProtocol string) BusType {
+	switch busProtocol {
+	case "PCI-Express":
+		return BUS_TYPE_NVME // is this even correct?
```

Comment:
hmm, from what I can tell, modern Macs use SATA drives. They used to use SCSI drives. Perhaps we need to rethink how we describe the "bus type" for block devices and instead indicate the storage interface (SATA vs SCSI, etc)... thoughts?

Reply:
I'm not sure, but I think my machine actually does use some (proprietary?) NVMe storage device. I'm only inferring this from the fact that it shows up under the "NVMExpress" section of "System Information", and [this article from a random search](https://beetstech.com/blog/apple-proprietary-ssd-ultimate-guide-to-specs-and-upgrades) seems to confirm.

Beyond that, I don't know, and this random new-ish MacBook Pro is the only Apple hardware I have to test on. My implementation of BusType is really just a best-effort to populate the field with _something_ for parity with the reports from Linux.